app/src/main/java/kr/co/connect/boostcamp/livewhere/firebase/ReviewDatabaseRepository.kt

```diff
@@ -0,0 +1,50 @@
+package kr.co.connect.boostcamp.livewhere.firebase
+
+import com.google.android.gms.tasks.Task
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
+import com.google.firebase.database.ValueEventListener
+import kr.co.connect.boostcamp.livewhere.model.Review
+import kr.co.connect.boostcamp.livewhere.model.entity.ReviewEntity
+
+
+abstract class ReviewDatabaseRepository<Entity, Model>(private val mapper: FirebaseMapper<Entity, Model>) {
```

Comment:
`p2;` 추상 클래스로 쓰일 것이라면 Base를 넣는 것이 좋아보여요.
```suggestion
abstract class BaseDatabaseRepository<Entity, Model>(private val mapper: FirebaseMapper<Entity, Model>) {
```